content/docs/studio/user-guide/projects-and-experiments/configure-a-project.md

```diff
@@ -8,61 +8,45 @@ may be required depending on how your Git repository has been set up.
 
 If you are connecting to a DVC repo which is at the root of the Git repository
 and does not reference remote/cloud storage, then you can successfully visualize
-it without configuring additional settings.
-
-If you are connecting to a non-DVC repository, if your metrics are in some
-custom files, if you are connecting to a monorepo, or if your metrics are in
-cloud or other remote storage, you will need to configure project settings.
-
-In each of these scenarios, you will need to configure additional settings for
-Iterative Studio to be able to access the data required for visualization.
-
-Additionally, you can also configure project settings to
-[change the name](#project-name) of your project and to
-[select columns](#columns) to import in your project.
-
-To go to project settings, click on the
-![](https://static.iterative.ai/img/studio/view_open_settings_icon.png) icon in
-the project. In the menu that opens up, click on `Settings`.
-
-![](https://static.iterative.ai/img/studio/project_open_settings.png)
+it without configuring additional settings. But in the following scenarios, you
+will need to configure project settings:
 
 ### Non-DVC repositories
 
-In the section on [preparing your repositories], you saw that you can use
-Iterative Studio with DVC as well as non-DVC repositories. If you are connecting
-to a non-DVC repository, then you will need to [specify the custom files] that
-contain the metrics and hyperparameters that you want to visualize.
+If you are creating a project for a non-DVC repository, you will need to
+[specify the custom files] that contain the metrics and hyperparameters that you
+want to visualize.
```

Comment:
I thought we were dropping this scenario?

Reply:
So far, we still have it.

Reply:
@tapadipti I think we have already committed to dropping it in https://github.com/iterative/studio/issues/5957 and have started taking action on it in https://github.com/iterative/studio/pull/6293. I don't see much downside to dropping it here.